examples/fabric-admin/main.cpp

```diff
@@ -28,6 +28,7 @@
 // ================================================================================
 int main(int argc, char * argv[])
 {
+    const char * args[] = { argv[0], "interactive", "start" };
```

Comment:
Since we hope to always start this app in interactive mode, so I want to see if we can save "interactive start"

Update the code to use command line to specify log path, like:
./fabric-admin --log-file-path /tmp/abc.log
or
./fabric-admin interactive start --logfile /tmp/fabric_admin.log

logs are redirected only when --log-file-path is specified.